packages/create-platformatic-db/src/create-package-json.mjs

```diff
@@ -0,0 +1,33 @@
+import pupa from 'pupa'
+import { isFileAccessible } from './utils.mjs'
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+const packageJsonTemplate = `\
+{
+  "name": "platformatic-db-api-example",
+  "version": "0.0.1",
+  "description": "Platformatic DB API",
+  "scripts": {
+    "start": "platformatic db start"
+  },
+  "devDependencies": {
+    "fastify": "^{fastifyVersion}",
```

Comment:
Is the `fastify` package installed so that the Fastify types are available in the generated plugin?